components/membership/registration-impl/src/main/kotlin/net/corda/membership/impl/registration/dynamic/mgm/MGMRegistrationMemberInfoHandler.kt

```diff
@@ -128,9 +132,10 @@ internal class MGMRegistrationMemberInfoHandler(
         }
     }
 
-    private fun buildMgmInfo(
+    fun buildMgmMemberInfo(
         holdingIdentity: HoldingIdentity,
-        context: Map<String, String>
+        context: Map<String, String>,
+        serialNumber: Long
```

Comment:
```suggestion
        serialNumber: Long,
```
Maybe we can also make this to have a default value.